test/cmdlineTests/ir_compiler_inheritance_nosubobjects/output

```diff
@@ -6,11 +6,18 @@ Optimized IR:
  *                !USE AT YOUR OWN RISK!               *
  *=====================================================*/
 
+/// @use-src 0:"ir_compiler_inheritance_nosubobjects/input.sol", 1:"#utility.yul"
 object "C_7" {
     code {
         {
+            /// @src 0:82:117
             mstore(64, 128)
-            if callvalue() { revert(0, 0) }
+            if /** @src 0:82:117 */ callvalue()
```

Comment:
This is the same source location as before - why is it repeated?

Reply:
And now I figured this out, too. In the `if` visitor, we process the body twice. One time just to check if it contains newlines, another time in the return expression. That completely confuses the order with the last location check...